src/features/authentication/login/login.scss

```diff
@@ -0,0 +1,181 @@
+.login {
+  .card-center {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+  }
+
+  &Title {
+    color: red;
+  }
+
+  &SomethingElse {
```

Comment:
Do you use the class `loginSomethingElse`? If not, then you could remove it. Also as a side note, I would recommend using more specific naming for classes so it's easier to understand.